### ISSUE TITLE
Per sample AUTC values

### ISSUE THREAD

**Issue by Hendrik-code** [BrainLesion/panoptica#123]
- Calculate the matching value in matching step, dont give final matching
- At the end, calculate the metrics for each threshold in range [0.0, 1.0] with default step-size 0.1 (or 0.05?)
- Then you can give an AUTC PER SAMPLE
- And with statistics, also averaged over all samples

- And then give matching arrays by function where you give the threshold it quickly computes it then.

**Comment by neuronflow** [comment]
I agree; having a convenient method for AUTC metrics would be amazing.
